### ISSUE TITLE
week 10 - day 01 - blog-o-sphere - 

### ISSUE THREAD

**Issue by BissellC** [BissellC/assignments#33]
# Blog Series Finally

We are approaching the final stretch of topics and tonight we will be preparing for the final stretch.

For this assignment, we are going to be blogging and practicing everything we have a talk about.

## Objectives

- Practice talking like a developer, by creating writing a blog about a topic that interests you. 
- Work on your capstone

## Explorer

- [ ] Blog time. Choose **one** prompt from the following list. Blogs should be of good quality and of decent length. The length should be long enough to get the message across.  [Medium](http://medium.com/) is a good blog creation site.  Turn in a link to your blog when you are finished. 

  - Pick 1 topic that we have learned and create a blog post explaining it to a new or non-developer
  - Blog about a struggle (technical or soft)  that you have encountered and overcame during your tenure here so far.
  - Pick a new technology that we haven't hit yet (or won't) and create an intro blog for that.

* [ ] Algorithm Practice -- submit a `gist` of your solution for https://www.codewars.com/kata/form-the-minimum in C#, and/or JavaScript

## Adventure

- [ ] Work on your capstone project

**Comment by BissellC** [comment]
**Overcoming fear of failure**

Quitting something before even having a chance to fail can be hard to overcome when someone is conditioned to think that way. I have experienced this in my life, and it took me a while to see what was happening. It's easy to protect my ego and just say "maybe I'm not cut out for this", but part of learning and growing is failure.

**My experience in public school and how it impacted me**

Before I joined elementary school my dad taught me how to read. In turn, things came easier for me, and teachers thought I might be some kind of genius. It was always the same for me, I did not have to put in effort to study because I was so far ahead. From there, I took "gifted" classes - if I could go back in time I never would have - and they sheltered me and stroked my ego, leading me to believe I had something nobody else had. My first academic struggle was in junior year of high school in pre-calculus. I failed the class after getting nearly all A's in other classes. I was very critical of myself, and after always imagining going to university I started to have second thoughts. How could I go into postsecondary education if I have never learned how to study? I ended going to community college for only one semester after taking a "gap year" and went on academic probation because I did the bare minimum.

I tend to shut down when I do not understand something, and I think that is because I am afraid of giving something my all and failing. Instead I give up before I even give something a proper chance. It is easier to quit before truly struggling, but what you learn through the struggle is what really sticks. It is embarrassing to have all of the opportunities of a white male in America and not give 100%, just because I let myself be lazy. 

**What I am learning today**

After a couple years of just getting by I yearned for a sense of purpose. That to become a man, I needed to have something to work towards every day. I decided to start working towards a career in computer programming at 22 years old. I am very excited to have a goal to work towards. My main motivator was to make money so I could be financially independent. I have now realized that it is going to take a lot more than wanting money to build a life for myself. Something inside me has to change.

When learning computer programming, failure is the name of the game. I started to learn basic web development on my own but became frustrated with the slow progress. Self motivation is not one of my strong suits, so I decided to attend a three month coding bootcamp for web development with Suncoast Developers Guild. With the pressure of the loan I took out and a great opportunity to start a career I dove into web development head first. I am currently 2 months into the course and I have grown as a developer and as a person. I have finally learned how to learn. I have been tempted to give up like my old self would, but after working with other people just like me trying to make it in the world it clicked with me that it is normal to struggle when learning something new. That every time I look at something I do not understand, things become a little clearer. You would think that I would have learned that by now in my academic career. It was hard for me to put my old habits aside to open myself up to something new. I am still not great at controlling my emotions about failure and fear of failure, but every day it becomes easier. I thought the biggest hurdle would be the actual coding, but for me it has been practicing mindfulness and creating new habits.
